libs/ledger-live-common/src/hooks/useAccountsWithFundsListener.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useMemo, useRef } from "react";
+import debounce from "lodash/debounce";
+import type { Account } from "@ledgerhq/types-live";
+
+export function hasAccountsWithFundsChanged(accounts: Account[], oldAccounts: Account[]): boolean {
```

Comment:
Wouldn't it be nice to add some comments/explanation about the function ? Like what do we actually mean by `changed` ?